virt/vm_templates/virt-automatic-bootsource-updates.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="virt-automatic-bootsource-updates"]
+= Managing automatic boot source updates
+include::_attributes/common-attributes.adoc[]
+:context: virt-automatic-bootsource-updates
+
+toc::[]
+
+By default, {VirtProductName} automatically updates system-defined boot sources. You can opt out of automatic updates by disabling the `enableCommonBootImageImport` feature gate.
+
+Custom boot sources are not controlled by the feature gate. You must manage them individually by editing the `HyperConverged` custom resource.
+
+[id="feature-gate_virt-automatic-bootsource-updates"]
+== Using the feature gate
+
```

Comment:
Looks much better, Pan! 

I have been giving some thought to the organization of this assembly. I think that organizing the contents as "custom vs system" might not be as clear as I had hoped. I wonder whether this would be clearer:

== Enabling boot source updates

=== Enabling updates for system boot sources by using the feature gate (currently "Toggling automatic updates for system-defined boot sources")

=== Enabling updates for custom boot sources by editing the HCO resource

(Prerequisite: Configuring storage class for custom boot sources, so that would have to be worked in somehow)

== Disabling boot source updates (currently: Disabling an automatic update for a system-defined or user-defined boot source)